### ISSUE TITLE
Accessibility: Toolbar navigation

### ISSUE THREAD

**Issue by alansemenov** [enonic/lib-admin-ui#3642]
* Should set focus on the first clickable control inside (filter panel icon)
* Navigation should be down with arrow keys (left/right)
* Tab should move focus to the next element on the page
